config.yml

```diff
@@ -14,6 +14,9 @@ enableRobotsTXT:        true
 metaDataFormat:         "yaml"
 disableKinds:           ["404", "taxonomy", "taxonomyTerm", "RSS"]
 
+blackfriday:
+  fractions:            false
+
```

Comment:
Why is this needed?

Reply:
From https://deploy-preview-28014--twbs-bootstrap.netlify.com/docs/4.3/getting-started/browsers-devices/, this:
> If you require Internet Explorer 10/11 support, use Bootstrap 4.

Was rendering as:
> If you require Internet Explorer <sup>10</sup>&#8260;<sub>11</sub> support, use Bootstrap 4.

Reply:
Ah, nice good catch. Makes sense then 👍